auth-api/src/auth_api/services/membership.py

```diff
@@ -69,26 +70,45 @@ def get_membership_type_by_code(type_code):
         return MembershipTypeModel.get_membership_type_by_code(type_code=type_code)
 
     @staticmethod
-    def get_members_for_org(org_id, status=None, membership_roles=None, token_info: Dict = None,
-                            allowed_roles: Tuple = None):
+    def get_members_for_org(org_id, status=Status.ACTIVE, \
+        membership_roles=ALL_ALLOWED_ROLES, token_info: Dict = None):  # pylint:disable=too-many-return-statements
         """Get members of org.Fetches using status and roles."""
-        if org_id is None:
-            return None
-
-        if membership_roles is None:
-            membership_roles = ALL_ALLOWED_ROLES
-        if not status:
-            status = Status.ACTIVE.value
-        else:
-            status = Status[status].value
-
         org_model = OrgModel.find_by_org_id(org_id)
         if not org_model:
             return None
 
-        # Check authorization for the user
-        check_auth(token_info, one_of_roles=allowed_roles, org_id=org_id)
-        return MembershipModel.find_members_by_org_id_by_status_by_roles(org_id, membership_roles, status)
+        status = Status.ACTIVE.value if status is None else Status[status].value
+        membership_roles = ALL_ALLOWED_ROLES if membership_roles is None else membership_roles
```

Comment:
This looks redundant, as there is default value on definition.

Reply:
Actually for both `status` and `membership_roles`, these can be explicitly passed as `None` from the org resource (if they are not included as query params).